src/structures/MessageEmbed.js

```diff
@@ -42,7 +42,7 @@ class MessageEmbed {
      * The color of this embed
      * @type {?number}
```

Comment:
This no longer only takes an integer with this change, but rather, `ColorResolvable`.

Reply:
As dim pointed out, the constructor isn't the recommended method of creating embeds, instead it seems the library leans towards using the `set` methods. The typings already account for the constructor, I'm not sure adding a typedef to it is worth it, as it will only clutter the documentation